src/ovh-angular-mondial-relay/ovh-angular-mondial-relay.directive.js

```diff
@@ -27,7 +27,7 @@
  * @param {string} [user-service='User'] The name of your user service, in case your service name is different from the default.
  */
 angular.module("ovh-angular-mondial-relay")
-    .directive("mondialRelay", function (MONDIAL_RELAY_PICS, MONDIAL_RELAY) {
+    .directive("mondialRelay", function (MONDIAL_RELAY_PICS, MONDIAL_RELAY, $translate) {
```

Comment:
prefer to order module this way : 

- Angular providers
- Services
- Constants

So for this case : `"mondialRelay", function ($translate, MONDIAL_RELAY_PICS, MONDIAL_RELAY`

Reply:
I guess you meant `($translate, MONDIAL_RELAY, MONDIAL_RELAY_PICS)` 😄